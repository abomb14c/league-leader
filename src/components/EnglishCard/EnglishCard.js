import React from 'react';

export const EnglishCard = (props) => {
  const {name, 
    wins,
    losses,
    draws,
    points,
    rank} = props; 

  return (
    <div>
      <h3>{name}</h3>
      <h3>{rank}</h3>
    </div>
  )
}

export default EnglishCard;