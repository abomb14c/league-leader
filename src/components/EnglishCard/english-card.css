.english-card {
  align-items: center;
  border-bottom: 1px solid #C63D0F;
  background-color: #3B3738;
  display: flex;
  height: 42px;
  padding-left: 2%;
}

.english-desc {
  font-weight: 200;
  color: #FDF3E7;
}

.english-title-container {
  text-align: left;
  width: 37%;
}

.english-rank {
  margin-left: 15%;
}

.english-record-container {
  display: flex;
  margin-left: 15%;
}

.english-points {
  margin-left: 15%;
}