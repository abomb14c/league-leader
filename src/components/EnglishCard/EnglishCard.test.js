import React from 'react';
import { shallow } from 'enzyme';
import  EnglishCard  from './EnglishCard';

describe('EnglishCard', () => {
  let wrapper;
  let mockProps; 

  beforeEach(() => {
    mockProps = {
      name: 'Arsenal',
      wins: 34,
      losses: 0,
      draws: 0,
      points: 100,
      rank: 1
    };

    wrapper = shallow(<EnglishCard
      {...mockProps}
    />);
  });
  
  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});