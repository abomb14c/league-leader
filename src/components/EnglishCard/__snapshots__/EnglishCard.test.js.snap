// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EnglishCard should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown
    draws={0}
    losses={0}
    name="Arsenal"
    points={100}
    rank={1}
    wins={34}
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="english-title-container"
>
          <h3
                    className="english-title english-desc"
          >
                    Arsenal
          </h3>
</div>,
        <h3
          className="english-rank english-desc"
>
          1
</h3>,
        <div
          className="english-record-container"
>
          <h3
                    className="english-wins english-desc"
          >
                    34
                    -
          </h3>
          <h3
                    className="english-losses english-desc"
          >
                    0
                    -
          </h3>
          <h3
                    className="english-draws english-desc"
          />
</div>,
        <h3
          className="english-points english-desc"
>
          100
</h3>,
      ],
      "className": "english-card",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <h3
            className="english-title english-desc"
>
            Arsenal
</h3>,
          "className": "english-title-container",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Arsenal",
            "className": "english-title english-desc",
          },
          "ref": null,
          "rendered": "Arsenal",
          "type": "h3",
        },
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": 1,
          "className": "english-rank english-desc",
        },
        "ref": null,
        "rendered": 1,
        "type": "h3",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h3
              className="english-wins english-desc"
>
              34
              -
</h3>,
            <h3
              className="english-losses english-desc"
>
              0
              -
</h3>,
            <h3
              className="english-draws english-desc"
/>,
          ],
          "className": "english-record-container",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                34,
                "-",
              ],
              "className": "english-wins english-desc",
            },
            "ref": null,
            "rendered": Array [
              34,
              "-",
            ],
            "type": "h3",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                0,
                "-",
              ],
              "className": "english-losses english-desc",
            },
            "ref": null,
            "rendered": Array [
              0,
              "-",
            ],
            "type": "h3",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": 0,
              "className": "english-draws english-desc",
            },
            "ref": null,
            "rendered": 0,
            "type": "h3",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": 100,
          "className": "english-points english-desc",
        },
        "ref": null,
        "rendered": 100,
        "type": "h3",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="english-title-container"
>
            <h3
                        className="english-title english-desc"
            >
                        Arsenal
            </h3>
</div>,
          <h3
            className="english-rank english-desc"
>
            1
</h3>,
          <div
            className="english-record-container"
>
            <h3
                        className="english-wins english-desc"
            >
                        34
                        -
            </h3>
            <h3
                        className="english-losses english-desc"
            >
                        0
                        -
            </h3>
            <h3
                        className="english-draws english-desc"
            />
</div>,
          <h3
            className="english-points english-desc"
>
            100
</h3>,
        ],
        "className": "english-card",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <h3
              className="english-title english-desc"
>
              Arsenal
</h3>,
            "className": "english-title-container",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Arsenal",
              "className": "english-title english-desc",
            },
            "ref": null,
            "rendered": "Arsenal",
            "type": "h3",
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": 1,
            "className": "english-rank english-desc",
          },
          "ref": null,
          "rendered": 1,
          "type": "h3",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h3
                className="english-wins english-desc"
>
                34
                -
</h3>,
              <h3
                className="english-losses english-desc"
>
                0
                -
</h3>,
              <h3
                className="english-draws english-desc"
/>,
            ],
            "className": "english-record-container",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  34,
                  "-",
                ],
                "className": "english-wins english-desc",
              },
              "ref": null,
              "rendered": Array [
                34,
                "-",
              ],
              "type": "h3",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  0,
                  "-",
                ],
                "className": "english-losses english-desc",
              },
              "ref": null,
              "rendered": Array [
                0,
                "-",
              ],
              "type": "h3",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": 0,
                "className": "english-draws english-desc",
              },
              "ref": null,
              "rendered": 0,
              "type": "h3",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": 100,
            "className": "english-points english-desc",
          },
          "ref": null,
          "rendered": 100,
          "type": "h3",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
